Makefile

```diff
@@ -382,7 +384,7 @@ endif
 WARNINGS = all error strict-prototypes missing-prototypes nested-externs \
 	missing-declarations undef pointer-arith no-nonnull declaration-after-statement
 
-CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding \
+CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding -fms-extensions \
```

Comment:
I would prefer not to add additional flags. Perfectly happy with the other changes though